src/components/languagechoices/LangItem.tsx

```diff
@@ -1,25 +1,25 @@
-/* eslint-disable react/prop-types */
 import React from 'react';
 import { Button } from 'react-bootstrap';
 import { useRecoilState } from 'recoil';
 import i18n from 'i18next';
 
+import { ILanguage } from 'Interfaces/ILanguage';
```

Comment:
This should be a global import using path aliases

Reply:
path aliases ain't working that's why i have not used in the interfaces as well